import_and_export_data/guides/create-connector.rst

```diff
@@ -315,3 +315,70 @@ If you don't see your changes, make sure you have run (``bin/console assets:inst
 Now that we have our extension loaded in our form, we can add some logic into it, check how to `customize the UI`_.
 
 .. _customize the UI: ../../design_pim
+
+Make a job stoppable
+--------------------
+
+As of now, it is possible your jobs won't be able to stop when the job is started.
+
+.. note::
+
+    If your job is based on the ``Akeneo\Tool\Component\Batch\Job\Job`` class, you only have to adapt your job service definition by setting the last property of the constructor to "true" (as shown in the example below).
```

Comment:
```suggestion
    If your job is based on the ``Akeneo\Tool\Component\Batch\Job\Job`` class, you have to adapt your job service definition by setting the last property of the constructor to "true" (as shown in the example below).
```

It's often advised to avoid "easy, only, simple" words as they may imply that something is easy or simple and could be frustrating if you are stuck